keps/sig-apps/85-Graduate-PDB-to-Stable/README.md

```diff
@@ -0,0 +1,319 @@
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+[Pod Disruption Budget (PDB)](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)
+is a Kubernetes API that limits the number of pods of a collection that are down simultaneously from voluntary disruptions.
+[Kubernetes eviction API](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#the-eviction-api)
+takes PDB into account when terminating pods. If PDB is
+violated, the eviction API returns failure and does not delete the requested pod.
+The feature was introduced in Kubernetes 1.4 and promoted to beta in 1.5.
+It has been in beta for a long time. This document lays out the plan to promote
+it to stable.
+
+## Motivation
+
+PDB API has been stable and is an important feature that allows users to improve
+reliability of their critical workloads. This feature has been in beta for a
+long time. We need to promote to stable version given that we plan to support it
+long term.
+
+### Goals
+
+* Plan to promote PDB API to stable version.
+* Address some open issues with PDBs
+
+## Proposal
+
+### Implementation Details/Notes/Constraints
+
+#### Mutable PDBs
+
+A mutable PDB object allows its `MinAvailable`, `MaxUnavailable`, and `Selector`
+fields to be modified by clients. Components that use PDB must watch such
+modifications and use the updated values when making decisions.
+
+This feature is implemented by [this PR](https://github.com/kubernetes/kubernetes/pull/69867).
+
+#### Eviction of non-ready pods
+
+There are a couple of open issues where pods can't be evicted, even if they not Ready and Running 
+(https://github.com/kubernetes/kubernetes/issues/72320 and https://github.com/kubernetes/kubernetes/issues/80389).
+The root of this issue, is that the rules in the disruption controller for what is a healthy pod, and the rules
```

Comment:
nit: can you please shorten the lines a bit - they are visibly longer in this section than in others

Reply:
Done